engine-tests/src/test/java/org/terasology/rendering/dag/RenderTaskListGeneratorTest.java

```diff
@@ -16,11 +16,22 @@
 package org.terasology.rendering.dag;
 
 import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
+import javafx.util.Pair;
```

Comment:
Is this intended, and do we really want to use JavaFX libraries in the main game? I have mixed feelings about this. On the one hand, the launcher uses it anyways, and with bundles JRE that should not be a problem.
On the other hand, JavaFX is not part of the default distribution of OpenJDK/JRE(?), and not everybody wants to/can install and use Oracle's Java. 

If `Pair` is the only thing we need from JavaFX, would it make sense to re-implement it and provide it as a standard type for Terasology?

Reply:
It shouldn't be there in the first place. I'll get rid of it. The method is not in use anyway, I haven't written tests for this since I didn't make changes to the RenderTaskListGenerator. Changes to RTLGTest were to compile so the file is just dragging along